Packages/nadena.dev.modular-avatar/Editor/MeshRetargeter.cs

```diff
@@ -165,8 +167,11 @@ public Mesh Retarget()
             avatarTransform.localScale = Vector3.one;
 
             src = renderer.sharedMesh;
-            dst = Mesh.Instantiate(src);
-            dst.name = "RETARGETED: " + src.name;
+            if (src != null)
```

Comment:
If src is null, why not just return immediately? The renderer is going to be broken with no mesh anyway.

Reply:
AvatarOptimizer will assign mesh later so I want MA to retarget bones (especially root transform & bounding box) of SkinnedMeshRenderer with null mesh.